src/components/main.jsx

```diff
@@ -122,6 +127,22 @@ export default class Main extends React.Component {
     if (this.state.tool === 'measure') {
       this.setState({ lengthMeasurePoints: [ ...this.state.lengthMeasurePoints, { lat, lon } ] });
     }
+    if(this.state.tool == 'route-planner'){
```

Comment:
Bolo by fajn mat konzistentny sposob kodu skrz celu appku. Navrhujem:
- pouzivat bodkociarky
- vzdy pouzivat bloky `{...}` za `if` / `for` / ...
- za klucovymi slovami davat vzdy medzeru (namiesto `if(...` pisat `if (...`)
- pouzivat striktne porovnavania ak nie je potrebne inac (`===` vs `==`)